wgsl/index.bs

```diff
@@ -5271,6 +5271,61 @@ A phony-assignment is useful for:
   </xmp>
 </div>
 
+## Increment and Decrement Statements ## {#increment-decrement}
+
+An <dfn noexport>increment statement</dfn> adds 1 to the contents of a variable.
+A <dfn noexport>decrement statement</dfn> subtracts 1 from the contents of a variable.
+
+<div class='syntax' noexport='true'>
+  <dfn for=syntax>increment_statement</dfn> :
+
+    | [=syntax/lhs_expression=] [=syntax/plus_plus=]
+</div>
+
+<div class='syntax' noexport='true'>
+  <dfn for=syntax>decrement_statement</dfn> :
+
+    | [=syntax/lhs_expression=] [=syntax/minus_minus=]
+</div>
+
+The expression must evaluate to a reference with an [=integer scalar=] [=store type=] and [=access/read_write=] [=access mode=].
+
+<table class='data'>
+  <thead>
+    <tr><th>Precondition<th>Statement<th>Description
```

Comment:
This is a lot of words.

Maybe take the approach in #2345, depending on feedback there.